File: dal-dao-gen/dal_ftp/63/cs/Entity/AccountBalanceLetter.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// AccountBalanceLetter
    /// </summary>
    [Serializable]
    [Table(Name = "AccountBalanceLetter")]
    public partial class AccountBalanceLetter
    {
        /// <summary>
        /// 获取或设置RecordId
        /// </summary>
        [Column(Name = "RecordId",Length=10),ID,PK]
        public int RecordId { get; set; }
        /// <summary>
        /// 获取或设置AccBalanceId
        /// </summary>
        [Column(Name = "AccBalanceId",Length=10)]
        public int? AccBalanceId { get; set; }
        /// <summary>
        /// 获取或设置FileName
        /// </summary>
        [Column(Name = "FileName",Length=500)]
        public string FileName { get; set; }
        /// <summary>
        /// 获取或设置UploadPerson
        /// </summary>
        [Column(Name = "UploadPerson",Length=500)]
        public string UploadPerson { get; set; }
        /// <summary>
        /// 获取或设置DescribeInfo
        /// </summary>
        [Column(Name = "DescribeInfo",Length=50)]
        public string DescribeInfo { get; set; }
        /// <summary>
        /// 获取或设置CreateTime
        /// </summary>
        [Column(Name = "CreateTime",Length=23)]
        public DateTime? CreateTime { get; set; }
        /// <summary>
        /// 获取或设置DataChangeLastTime
        /// </summary>
        [Column(Name = "DataChangeLastTime",Length=23)]
        public DateTime? DataChangeLastTime { get; set; }
    }
}

File: dal-dao-gen/dal_ftp/63/java/Entity/AccountBalanceLetter.java
package com.ctrip.dal.test.test4;

import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class AccountBalanceLetter implements DalPojo {
	private Integer recordId;
	private Integer accBalanceId;
	private String fileName;
	private String uploadPerson;
	private String describeInfo;
	private Timestamp createTime;
	private Timestamp datachangeLasttime;
	public Integer getRecordId() {
		return recordId;
	}

	public void setRecordId(Integer recordId) {
		this.recordId = recordId;
	}

	public Integer getAccBalanceId() {
		return accBalanceId;
	}

	public void setAccBalanceId(Integer accBalanceId) {
		this.accBalanceId = accBalanceId;
	}

	public String getFileName() {
		return fileName;
	}

	public void setFileName(String fileName) {
		this.fileName = fileName;
	}

	public String getUploadPerson() {
		return uploadPerson;
	}

	public void setUploadPerson(String uploadPerson) {
		this.uploadPerson = uploadPerson;
	}

	public String getDescribeInfo() {
		return describeInfo;
	}

	public void setDescribeInfo(String describeInfo) {
		this.describeInfo = describeInfo;
	}

	public Timestamp getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Timestamp createTime) {
		this.createTime = createTime;
	}

	public Timestamp getDatachangeLasttime() {
		return datachangeLasttime;
	}

	public void setDatachangeLasttime(Timestamp datachangeLasttime) {
		this.datachangeLasttime = datachangeLasttime;
	}

}